src/Relation/Binary/Bundles/Raw.agda

```diff
@@ -0,0 +1,118 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Raw bundles for homogeneous binary relations
+------------------------------------------------------------------------
+
+-- The contents of this module should be accessed via `Relation.Binary`.
+
+{-# OPTIONS --cubical-compatible --safe #-}
+
+module Relation.Binary.Bundles.Raw where
+
+open import Function.Base using (flip)
+open import Level using (Level; suc; _⊔_)
+open import Relation.Binary.Core using (Rel)
+open import Relation.Nullary.Negation.Core using (¬_)
+
+
+------------------------------------------------------------------------
+-- RawSetoid
+------------------------------------------------------------------------
+
+record RawSetoid a ℓ : Set (suc (a ⊔ ℓ)) where
+  infix 4 _≈_
+  field
+    Carrier              : Set a
+    _≈_                  : Rel Carrier ℓ
+
+  infix 4 _≉_
+  _≉_ : Rel Carrier _
+  x ≉ y = ¬ (x ≈ y)
+
+
+------------------------------------------------------------------------
+-- RawPreorder
+------------------------------------------------------------------------
+
+record RawPreorder c ℓ₁ ℓ₂ : Set (suc (c ⊔ ℓ₁ ⊔ ℓ₂)) where
```

Comment:
Hmm so following the pattern of `Algebra.Bundles.Raw`, then bundle equivalence is determined not by the names of the relation, but by their signature. For example both `CommutativeMonoid` and `Monoid` both have a single `RawMonoid` base.

Therefore to be consistent I think `RawPreorder`, `RawPartialOrder`, `RawStrictPartialOrder` and `RawApartness` relation should all be a single record... With what name I don't know...

If the purpose of this is to introduce syntax, then I think we need to use some other mechanism other than `Raw` bundles...

Reply:
Two things:
* good point about shared signatures (and `\~` would be the 'canonical' choice if symbol here?), and then let the corresponding `Raw`sub bundles in `Bundles` handle the renaming to orthodox/rectified syntax?
* as with discussions with @JacquesCarette , for me the issue is one if being able, for a given raw signature, to enrich it with 'obvious' derived operations which allow smoother treatment of subsequent properties etc. The 'syntax' aspect here is really about standardising notation for these otherwise useful/important derived notions...?